src/components/dev-dot-content/mdx-components/mdx-code-blocks/index.tsx

```diff
@@ -70,12 +70,12 @@ function CodeBlockConfig({
 	hideClipboard,
 }: CodeBlockConfigProps) {
 	// This is only used for debugging purposes as we use shiki to assign the language tokens and styles
-	const propsOfFristChild = propsOfFirstChild(children)
```

Comment:
Renamed this function so I could fix the typo in the `const` name (otherwise it'd be a duplicate declaration).

Note with the modifications to the function, this may end up being `null`. This case was already handled - the `language` prop we pass may end up being `null`, which is not ideal, but doesn't seem worth stopping the whole page over.